src/components/KlokkeslettField.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import MaskedInput from "react-maskedinput";
+import { SkjemaelementFeilmelding } from "nav-frontend-skjema";
+import { FieldInputProps, FieldMetaState } from "react-final-form";
+
+interface KlokkeslettFieldProps {
+  meta: FieldMetaState<string>;
+  id: string;
+  input: FieldInputProps<string, HTMLInputElement>;
+  type?: string;
+  className?: string;
+  placeholder?: string;
+}
+
+const KlokkeslettField = ({
+  meta: { touched, error },
+  input,
+  id,
+  type = "text",
+  className,
+  placeholder,
+}: KlokkeslettFieldProps) => (
+  <>
+    <MaskedInput
```

Comment:
Hva er årsaken til at man bruker maskering på klokkeslett? Og et maskeringsbibliotek?

Reply:
🤷‍♂️ Har hovedsakelig bare konvertert fra js til tsx her og fant ingen ferdig komponent for dette i designsystemet, så da lot jeg det være slik. Kan sikkert fikse det i en ny PR hvis du har en bedre løsning :)